include/openenclave/edl/sgx/platform.edl

```diff
@@ -19,4 +19,7 @@ enclave
     from "openenclave/edl/sgx/debug.edl" import *;
     from "openenclave/edl/sgx/thread.edl" import *;
     from "openenclave/edl/sgx/switchless.edl" import *;
+#ifdef OE_SGX_ENTROPY
+    from "openenclave/edl/sgx/entropy.edl" import *;
+#endif
```

Comment:
Isn't OE_SGX compiler flag accessible here. Trying to understand why we introduced new compiler variable OE_SGX_ENTROPY for sgx platform file.

Reply:
The purpose of the new flag is preventing including the `entropy.edl` by default along with `platform.edl`. The behavior this PR wants to enforce is that the developer should explicitly opt into the `entropy.edl` to use the FIPS module. However, internally we can supply the `OE_SGX_ENTROPY` variable for forward declaration so that we don't need an additional invocation of the `oeedger8r`.